### PR TITLE
MINOR: Copy only client jars

### PR DESCRIPTION
**Pull request by enothereska**
@granders @ewencp @miguno : have a look when you can. I was copying more jars than necessary. Like in packaging I only copy the s-m-client jars. Also changed from dev to package (that was not the source of the problem though)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -172,8 +172,8 @@ if [ "$PS_ENABLED" = "yes" ]; then
     build_maven_project "support-metrics-client" "${GIT_MODE}confluentinc/support-metrics-client.git" "clean package install"
 
     # Copy support-metrics-* jars to kafka
-    for library in "support-metrics-client" "support-metrics-common" "support-metrics-fullcollector"; do
-        for file in $projects_dir/$library/package/target/support-metrics-*-package-*-development/share/java/confluent-support-metrics*/*.jar; do
+    for library in "support-metrics-client"; do
+        for file in $projects_dir/$library/package/target/support-metrics-*-package-*-package/share/java/confluent-support-metrics*/*.jar; do
```
Comment (miguno) [inline build.sh]: I think a more precise glob pattern is:

```bash
for file in $projects_dir/$library/package/target/support-metrics-client-package-*-package/share/java/confluent-support-metrics-client/*.jar; do
```

**Comment by miguno** [comment]
Added one comment, otherwise LGTM.  (Caveat: I don't know a whole lot about muckrake.)

**Comment by ewencp** [comment]
@enothereska If this fixes PS, do you also want to toggle `PS_ENABLED` back to the default `"yes"` in this PR?

**Comment by enothereska** [comment]
@ewencp : done, thanks.

**Pull request merged**